src/cmake/config/win-vs.txt

```diff
@@ -154,18 +154,6 @@ if (USE_AVX2)
         /arch:AVX2                          # Advanced Vector Extensions 2
     )
 endif ()
-if (USE_FMA)
```

Comment:
> Maybe we could hide this options for windows?
It's not because the compiler doesn't explicitely support this architecture that we can't write specialized code using instrinsics, so I would keep exposing this option, even on Windows.